setup/sql/dbupdate_05.php

```diff
@@ -1103,22 +1103,22 @@
 <#5505>
 <?php
 if ($ilDB->tableExists('il_bt_value')) {
-    $deleteBucketsSql = 'DELETE FROM il_bt_value WHERE bucket_id IN (SELECT id FROM il_bt_bucket WHERE title = "Certificate Migration")';
+    $deleteBucketsSql = 'DELETE FROM il_bt_value WHERE bucket_id IN (SELECT id FROM il_bt_bucket WHERE title = ' . $ilDB->quote('Certificate Migration', 'text') . ') ';
     $ilDB->manipulate($deleteBucketsSql);
 }
 
 if ($ilDB->tableExists('il_bt_value_to_task')) {
-    $deleteValueToTask = 'DELETE FROM il_bt_value_to_task WHERE bucket_id IN (SELECT id FROM il_bt_bucket WHERE title = "Certificate Migration")';
+    $deleteValueToTask = 'DELETE FROM il_bt_value_to_task WHERE bucket_id IN (SELECT id FROM il_bt_bucket WHERE title =  ' . $ilDB->quote('Certificate Migration', 'text') . ')';
     $ilDB->manipulate($deleteValueToTask);
 }
 
 if ($ilDB->tableExists('il_bt_task')) {
```

Comment:
I am not sure if `ilCertificateMigrationReducedInteraction` is sufficient here. We also had `ilCertificateMigrationInteraction` and `ilCertificateMigrationJob`.

Reply:
`LIKE "ilCertificateMigration%"`?